documentation/Installation.md

```diff
@@ -0,0 +1,85 @@
+# Installing Dynamic Extensions in Alfresco
+
+Dynamic Extensions (DE) is distributed as an Alfresco Module Package (AMP) extension that can be installed in Alfresco.
+
+## Installing the DE AMP
+
+To support multiple Alfresco versions, different AMP's for each minor Alfresco version update are 
+build and distributed. E.g. if you are working with Alfresco 6.0.7-ga, you should
+use the `alfresco-dynamic-extensions-repo-60` artifact.
+
+### Maven Central Coordinates
+All required artifacts, including the AMP to be installed in Alfresco, are available in Maven Central.
+
+```xml
+<dependency>
+    <groupId>eu.xenit</groupId>
+    <artifactId>alfresco-dynamic-extensions-repo-${alfresco-version}</artifactId>
+    <version>${latest-dynamic-extensions-version}</version>
+    <type>amp</type>
+</dependency>
+```
+
+```groovy
+alfrescoAmp "eu.xenit:alfresco-dynamic-extensions-repo-${alfrescoVersion}:${latest-dynamic-extensions-version}@amp"
+
+```
+
+These artifacts can be used to automatically install DE in Alfresco using e.g. the Alfresco Maven SDK or 
+the [Alfresco Gradle Docker Plugins](https://github.com/xenit-eu/alfresco-docker-gradle-plugin)
+
+### Manual download and install
+
+* Download the latest <a href="https://github.com/xenit-eu/dynamic-extensions-for-alfresco/releases">Dynamic Extensions AMP</a>.
+* Use the <a href="https://docs.alfresco.com/6.1/concepts/dev-extensions-modules-management-tool.html">Module Management Tool</a> to install the AMP in the Alfresco repository of your choosing.
+* After restarting Alfresco, open the Control Panel: <a href="http://localhost:8080/alfresco/service/dynamic-extensions/">http://localhost:8080/alfresco/service/dynamic-extensions/</a>.
+* Accessing the Control Panel requires an admin account.
+
+## Supported Alfresco versions
+
+Dynamic Extensions is systematically integration tested against:
```

Comment:
integration-tested

Reply:
Actually I wanted to remove this file, since it is already in the README?

Reply:
Fine by me